application/src/main/java/org/thingsboard/server/actors/ruleChain/DefaultTbContext.java

```diff
@@ -161,8 +162,8 @@ public void enqueue(TbMsg tbMsg, Runnable onSuccess, Consumer<Throwable> onFailu
     }
 
     @Override
-    public void enqueue(TbMsg tbMsg, String queueName, Runnable onSuccess, Consumer<Throwable> onFailure) {
-        TopicPartitionInfo tpi = resolvePartition(tbMsg, queueName);
+    public void enqueue(TbMsg tbMsg, QueueId queueId, Runnable onSuccess, Consumer<Throwable> onFailure) {
```

Comment:
Please ensure backward compatibility and leave old methods with @Deprecated annotation.

Reply:
You may add log.warn on execution of deprecated method.